config/routes.rb

```diff
@@ -315,31 +314,6 @@
       get '/cac/:step' => 'cac#show', as: :cac_step
       put '/cac/:step' => 'cac#update'
     end
-    if Figaro.env.enable_mobile_capture == 'true'
-      get '/verify/doc_auth_v2' => 'idv/doc_auth_v2#index'
-      get '/verify/doc_auth_v2/scan_id' => 'idv/scan_id#new'
-      get '/verify/doc_auth_v2/:step' => 'idv/doc_auth_v2#show', as: :idv_doc_auth_v2_step
-      put '/verify/doc_auth_v2/:step' => 'idv/doc_auth_v2#update'
-      get '/verify/doc_auth_v2/link_sent/poll' => 'idv/doc_auth_v2#doc_capture_poll'
-      get '/verify/doc-auth-v2/:step' => 'idv/doc_auth_v2#show',
-          # sometimes underscores get messed up when linked to via SMS
-          as: :idv_doc_auth_v2_step_dashes
```

Comment:
this URL is referenced in the `send_link_step` so I think we can remove it from there now